yt_dlp/extractor/cbc.py

```diff
@@ -228,6 +229,63 @@ def _real_extract(self, url):
         }
 
 
+class CBCPlayerPlaylistIE(InfoExtractor):
+    IE_NAME = 'cbc.ca:player:playlist'
+    _VALID_URL = r'(?:cbcplayer:|https?://(?:www\.)?cbc\.ca/(?:player/)(?!play/))(?P<id>.+)'
```

Comment:
- `cbcplayer:` prefix should only be matched by `CBCPlayerIE`
 - `.+` is too broad, we want to exclude query and fragment, yeah?
```suggestion
    _VALID_URL = r'https?://(?:www\.)?cbc\.ca/(?:player/)(?!play/)(?P<id>[^?#]+)'
```